src/diffusers/schedulers/scheduling_euler_discrete.py

```diff
@@ -18,6 +18,7 @@
 
 import numpy as np
 import torch
+from k_diffusion.sampling import get_sigmas_karras
```

Comment:
We can import from the `k_diffusion` library here sadly. Could you just copy-paste the function here: https://github.com/crowsonkb/k-diffusion/blob/b43db16749d51055f813255eea2fdf1def801919/k_diffusion/sampling.py#L17 into the scheduler (ideally with a note that it's copied from the library to correctly cite)

Reply:
Original commit lgtm